File: src/main/java/com/prasad/nithin/xmas/entity/Results.java
/**
 * 
 */
package com.prasad.nithin.xmas.entity;

import java.util.List;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

/**
 * @author 982168
 *
 */
@Data
@AllArgsConstructor
@NoArgsConstructor
public class Results {

	private Question question;
	
	private List<UserResult> results;
	
	private int totalVotes;
	
	private int totalUniqueVotes;
	
}

File: src/main/ng-app/xmas-ui/src/app/datamodel/Results.ts
import { Question } from "./Question";
import { UserResult } from "./UserResult";

export class Results{
    
    constructor(private _question:Question,private _results:Array<UserResult>,private _totalVotes:Number,private _totalUniqueVotes:Number){
        
    }
    get question(){
        return this._question;
    }
    get results(){
        return this._results;
    }
    get totalVotes(){
        return this._totalVotes;
    }
    get totalUniqueVotes(){
        return this._totalUniqueVotes;
    }


}